zenpy/lib/mapping.py

```diff
@@ -87,7 +87,8 @@ class ZendeskObjectMapping(object):
         'recipient_address': RecipientAddress,
         'recipient': Recipient,
         'response': Response,
-        'trigger': zenpy.lib.api_objects.Trigger
+        'trigger': zenpy.lib.api_objects.Trigger,
+        'dynamic_content_item': DynamicContent
```

Comment:
If you call this "item", you don't need to implement a request/response handler or a generator. It should all just work. I think it is better to follow the pattern of calling the object what Zendesk does where possible.